CHANGES.md

```diff
@@ -1,3 +1,33 @@
+### 2.1.0 (2019-07-10)
+
+- Move to the last version of `decompress` (@dinosaure, #366)
+- Check order of entries in a tree object (bug found by @samoht, fixed by @dinosaure, #365)
+- Use `mmap` package (@dinosaure, #347, #360)
+- Update README.md (@tcoopman, @dinosaure, #337, #359)
+- `trim` the window used to pack (@pqwy, @dinosaure, #357, #358)
+- Use lastest version of `lru.0.3.0` (@pqwy, @dinosaure, #352, #356)
+- Fix smart protocol (fixed by @clecat and @dinosaure, feedbacks from @hannesm)
+ * Pull-request #351, #350, #338
+ * Issues #335, #342, #346
+ 
+   + regression tests was added (@dinosaure)
+   + semantics about negociation was explained (@clecat)
+   + end-to-end tests was done (@hannesm)
```

Comment:
FWIW, the end-to-end test is not yet satisfactory (see https://github.com/mirage/ocaml-git/issues/364 which shows an issue of the TCP-smart implementation)